dpanonymize/surveys/__init__.py

```diff
@@ -199,18 +196,28 @@ def load_raw_return_proc_csv(csv_loc: str,
     return raw_df_subject
 
 
-def process_and_copy_db(Lochness, subject, raw_input, proc_dst):
-    '''Process PII and copy the json to GENERAL/surveys/processed'''
-    pii_table_loc = get_PII_table_loc(Lochness, subject.study)
+def process_and_copy_db(pii_table_loc: Union[str, Path], subject_name: str,
+                        raw_input: Union[str, Path],
+                        proc_dst: Union[str, Path]):
+    '''Process PII and copy the json to GENERAL/surveys/processed
 
+    Key Arguments:
+        pii_table_loc: path of the pii table, str or Path.
+        subject_name: subject name to be used in the string replacement, str.
+        raw_input: raw data in, str or Path.
+        proc_dst: processed data out destination, str or Path.
+    '''
     # don't run this if the pii_table in the config.yml is missing
-    if pii_table_loc != False and pii_table_loc != '':
+    if pii_table_loc is not False and pii_table_loc != '':
```

Comment:
Doesn't just `if pii_table_loc:` work?

Reply:
At the very least, it could be abridged as `if not pii_table_loc and pii_table_loc != ' ':`

Reply:
Thanks for this. Simplified here https://github.com/AMP-SCZ/dpanonymize/pull/13/commits/19edc4e59fd30be930721d0b699ca4690cc83064